native_client/javascript/Makefile

```diff
@@ -17,19 +18,21 @@ clean:
 	rm -rf ./build/
 
 clean-npm-pack:
-	rm -fr ./node_modules/
 	rm -fr ./deepspeech-*.tgz
 
 really-clean: clean clean-npm-pack
+	rm -fr ./node_modules/
 	rm -fr ./lib/
 
 package.json: package.json.in
 	sed \
 		-e 's/$$(PROJECT_NAME)/$(PROJECT_NAME)/' \
 		-e 's/$$(PROJECT_VERSION)/$(PROJECT_VERSION)/' \
 		package.json.in > package.json && cat package.json
+npm-dev:
+	npm install --prefix=$(NPM_ROOT)/../ --ignore-scripts .
```

Comment:
We can't `--only=dev` because then we have only `devDependencies` of our depending packages. And it's not enough to run.